opentelemetry-http/README.md

```diff
@@ -1,37 +1,30 @@
+# OpenTelemetry HTTP
+
 ![OpenTelemetry — An observability framework for cloud-native software.][splash]
 
 [splash]: https://raw.githubusercontent.com/open-telemetry/opentelemetry-rust/main/assets/logo-text.png
 
-# OpenTelemetry HTTP
-
-The Rust [OpenTelemetry](https://opentelemetry.io/) implementation.
+This crate contains helper implementations for sending HTTP requests. Uses
+include propagating and extracting context over http, exporting telemetry,
+requesting sampling strategies.
```

Comment:
Something like - 

This crate facilitates HTTP header propagation and provides a trait-based HTTP client abstraction for telemetry export using OpenTelemetry. It includes utilities for injecting and extracting headers and supports various HTTP clients such as `reqwest`, `isahc`, and `hyper`.

Reply:
> provides a trait-based HTTP client abstraction for telemetry export using OpenTelemetry

Honestly we probably should note this won't be needed after http 1.0 upgrade

Reply:
yes good point.